app/src/test/java/org/mozilla/focus/telemetry/TelemetryIntegrationTest.kt

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.telemetry
+
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.times
+import org.mockito.Mockito.verify
+import org.mozilla.focus.FocusApplication
+import org.mozilla.focus.utils.anyNonNull
+import org.mozilla.telemetry.Telemetry
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+
+class NoTelemetryApplication : FocusApplication() {
+    override fun initTelemetry() {}
+}
+
+@RunWith(RobolectricTestRunner::class)
+@Config(application = NoTelemetryApplication::class)
```

Comment:
I realized that robolectric could, but highly improbably, be uploading telemetry data: I filed https://github.com/mozilla-mobile/firefox-tv/issues/1300